.github/workflows/infrastructure.yml

```diff
@@ -1058,3 +1078,84 @@ jobs:
             --auto-mitigate true \
             --severity 0 \
             --tags ${{ env.TAGS }}
+
+  # 8. Custom domains
+  domains:
+    name: Domains 🌐
+    needs: [base, security, webapp, dns]
+    environment: ${{ needs.base.outputs.environment }}
+    runs-on: [self-hosted, EXIP, infrastructure]
+    steps:
+      - name: Pre-production 💫
+        if: contains('["dev", "feature", "staging"]', env.TARGET)
+        run: echo "TYPE=Preproduction" >> $GITHUB_ENV
+
+      - name: Production 💫
+        if: ${{ 'production' == env.TARGET }}
+        run: echo "TYPE=Production" >> $GITHUB_ENV
+
+      - name: Tags 🏷️
+        run: echo TAGS='Environment=${{ env.TYPE }}' \
+          'Product=${{ env.PRODUCT }}' \
+          'Team=development' >> $GITHUB_ENV
+
+      - name: Login 🔐
+        uses: azure/login@v2
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Azure defaults ✨
+        uses: azure/cli@v2
+        with:
+          inlineScript: |
+            # Basic
+            az configure --defaults location=${{ vars.REGION }}
+            az configure --defaults group=rg-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}
+
+      - name: Certificates 📇
+        uses: azure/cli@v2
+        with:
+          inlineScript: |
+            az afd secret create \
+            --name kv-certificate-quote-latest \
+            --profile-name frontdoor-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }} \
+            --use-latest-version true \
+            --secret-source $(az keyvault list --query '[].id' -o tsv)/secrets/kv-certificate-quote-latest
+
+            az afd secret create \
+            --name kv-certificate-apply-latest \
+            --profile-name frontdoor-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }} \
+            --use-latest-version true \
+            --secret-source $(az keyvault list --query '[].id' -o tsv)/secrets/kv-certificate-apply-latest
+
+      - name: Domains 🌐
```

Comment:
we have 2 Domains as name, could one be a different name?

Reply:
Have updated.